src/v2/styles/AttachmentList/AttachmentList-theme.scss

```diff
@@ -315,7 +344,11 @@
   }
 
   .str-chat__message-attachment-audio-widget--progress-track {
-    background-color: var(--str-chat__disabled-color);
+    background: linear-gradient(
```

Comment:
@szuperaz I have not found use of `str-chat__message-attachment-audio-widget--progress-track` class in the Angular SDK, so should not impact it.